Source/WebCore/page/ShareData.h

```diff
@@ -34,7 +34,7 @@ struct ShareData {
     String title;
     String text;
     String url;
-    Vector<RefPtr<File>> files;
+    Vector<RefPtr<File>> files = { };
```

Comment:
This change is suspicious.  Is it really needed?  It shouldn't be.  The '=' looks especially unneeded.

Reply:
I previously got a warning/error because we're compiling with -Wmissing-field-initializers and constructing the struct from the decode routine without initialising the files field (it looks like an additional field which wasn't ever added to the original encode/decoders).

Would you prefer:
Vector<RefPtr<File>> files { };
or a different approach?

Reply:
Oh, I see now.  files is needed for the JS API but it is not serialized.  This maintains existing behavior.
Removing the = would be a slight improvement.
Reorganizing the serialization so we don't have a partially serialized struct would be a bigger improvement.  Not sure how much work that is or if it's worth it.